app/core/models/ocelot/stanzas/CalculationStanza.scala

```diff
@@ -120,26 +134,33 @@ sealed trait Operation {
 
 case class AddOperation(left: String, right: String, label: String) extends Operation {
 
-  def eval(labels: Labels): Labels = op(_ + _, addString, labels)
+  def eval(labels: Labels): Labels = op(_ + _, addString, unsupportedOperation, labels)
 
 }
 
 case class SubtractOperation(left: String, right: String, label: String) extends Operation {
 
-  def eval(labels: Labels): Labels = op(_ - _, unsupportedOperation, labels)
+  def eval(labels: Labels): Labels = op(_ - _, unsupportedOperation, subtractDate, labels)
+
+  private def subtractDate(date: LocalDate, other: LocalDate) : Option[String] = {
+
+    val days: Long = other.until(date, ChronoUnit.DAYS)
+
+    Try(days.toString).map(s => s).toOption
+  }
 
 }
 
 case class CeilingOperation(left: String, right: String, label: String) extends Operation {
 
-  def eval(labels: Labels) : Labels = rounding(ceiling, labels)
+  def eval(labels: Labels): Labels = rounding(ceiling, labels)
 
   private def ceiling(value: BigDecimal, scale: Int): BigDecimal = value.setScale(scale, RoundingMode.CEILING)
```

Comment:
dedicated fn not needed, can use `_.setScale(_, RoundingMode.CEILING)` in call to rounding()